src/config-loader/env.ts

```diff
@@ -112,6 +112,11 @@ export const db = {
 	 * Defaults to true when in DEBUG mode, false otherwise
 	 */
 	checkReadOnlyQueries: DEBUG,
+	/**
+	 * Prepare the statement after N executions, false to never prepare, true to always prepare, otherwise a positive number
+	 * Currently only supported for postgres
+	 */
+	prepareAfterN: false as boolean | number,
```

Comment:
Why is this typed as `false as boolean | number`  when the comment leads me to believe that `true` is a valid type?

Reply:
`true` is valid, that's why we have the cast to make the typing include it

Reply:
That is the type @otaviojacobi, this is a literal object with defaults, with an `as boolean | number` to avoid the type being inferred as `false` from the default and instead use the expected `boolean | number` for the type of the object whilst still initializing with the default value